lib/cache/cache.go

```diff
@@ -370,7 +369,7 @@ func ForOkta(cfg Config) Config {
 	cfg.target = "okta"
 	cfg.Watches = []types.WatchKind{
 		{Kind: types.KindUser},
-		{Kind: types.KindAppServer},
+		{Kind: types.KindAppServer, SubKind: types.SubKindOktaApp},
```

Comment:
Why does the okta service need a cached view over the okta appservers?

Reply:
I'll excise this -- I think the cache/event modifications were overkill.